pom.xml

```diff
@@ -56,7 +56,7 @@
 			<dependency>
 				<groupId>com.fasterxml.jackson.core</groupId>
 				<artifactId>jackson-databind</artifactId>
-				<version>2.13.4.1</version>
+				<version>2.14.0-rc1</version>
```

Comment:
We should remove this versioning altogether and let Spring Boot dictate the version. Can you remove the version and see what it brings?